drivers/dose/dose.c

```diff
@@ -669,4 +771,14 @@ void dose_setup(dose_t *ctx, const dose_params_t *params, uint8_t index)
     DEBUG("dose timeout set to %" PRIu32 " µs\n", ctx->timeout_base);
     ctx->timeout.callback = _isr_xtimer;
     ctx->timeout.arg = ctx;
+
+#ifdef MODULE_DOSE_WATCHDOG
```

Comment:
Can you use `IS_USED`?

Reply:
Well the driver doesn't use `IS_USED` yet so I found `#ifdef` to be more consistent.